### DISCUSSION TITLE
Displaying existing issues while typing a new issue

### DISCUSSION THREAD

**Discussion by Monstarrrr** [community/community#135930] [Projects and Issues]
### Select Topic Area

General

### Feature Area

Issues

### Body

Sometimes users create issues that already exist, and that could be prevented using a search feature within the issue form title.  
As a user types the issue, a dropdown shows up below with 2 or 3 issues matching keywords from that title.  
User can click on it to go to that issue.

![image](https://github.com/user-attachments/assets/b681b872-77e2-4ded-abab-bfbc76f01112)

I know in theory a user is supposed to do a search before submitting an issue, but sometimes I can find myself forgetting I actually had created a specific issue a while ago so I won't think of searching if it exists first. And let's be honest some people still create issues in public repos that have already been created by someone else and they just don't check.  

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by ShaunaGordon** [comment]
Ironically, this is a duplicate -- #8277 -- which really proves the point of the need for this.
